### PR TITLE
fix(ui): Fix project selector menu

### PR DESCRIPTION
**Pull request by billyvg**
Fixes project selector dropdown menu, fixes the height of global selection header items

### CODE REVIEW

**Review by markstory — APPROVED** [review]
Looks good to me.

File: src/sentry/static/sentry/app/components/organizations/globalSelectionHeader/globalSelectionHeader.jsx [hunk 1]
```diff
@@ -321,8 +326,7 @@ class GlobalSelectionHeader extends React.Component {
       <Header className={className}>
         <HeaderItemPosition>
           {forceProject && this.getBackButton()}
```
Comment (Chrissy) [inline src/sentry/static/sentry/app/components/organizations/globalSelectionHeader/globalSelectionHeader.jsx]: can this not exist directly under `Header` now? That way it won't affect the layout of the first `HeaderItem` and cause it to be different.

**Review by Chrissy — APPROVED** [review]
One comment which we can address in a follow up

File: src/sentry/static/sentry/app/components/organizations/globalSelectionHeader/globalSelectionHeader.jsx (hunk 1 above)
Comment (billyvg) [reply src/sentry/static/sentry/app/components/organizations/globalSelectionHeader/globalSelectionHeader.jsx]: Yeah, would you mind doing this in a follow up?

**Pull request merged**